AUTHORS

```diff
@@ -2,146 +2,147 @@ The django-import-export was original created by Bojan Mihelac (bmihelac).
+-   jbub (Juraj Bubniak)
+-   manelclos (Manel Clos)
+-   pkozlov (Pavel Kozlov)
+-   vparitskiy
+-   snj (KINOSHITA Shinji)
+-   AyumuKasuga (Andrey Kostakov)
+-   luto
+-   gallochri (Christian Galeffi)
+-   ericdwang (Eric Wang)
+-   gonzalobustos (Gonzalo Bustos)
+-   domenkozar (Domen Kožar)
+-   staranjeet (Taranjeet Singh)
+-   mikhailmain
+-   retrry (Tadas Barzdžius)
+-   ericbuckley (Eric Buckley)
+-   PetrDlouhy
+-   shalakhin (Olexandr Shalakhin)
+-   fabian (Fabian Vogler)
+-   citmusa (Citlalli Murillo)
+-   jarekwg (Jarek Glowacki)
+-   fossilet (Yongzhi Pan)
+-   jnns (Jannis)
+-   sgarcialaguna
+-   carlosp420 (Carlos Peña)
+-   freelancersunion (Freelancers Union)
+-   jbradberry (Jeff Bradberry)
+-   antoniablair
+-   kellerkind
+-   yueyoum (Johnnie Wang)
+-   shaggyfrog (Thomas Hauk)
+-   suriya (Suriya Subramanian)
+-   arj03 (Anders Rune Jensen)
+-   arseniy-panfilov (Arseniy Panfilov)
+-   jheld (Jason Held)
+-   pajod (Paul J. Dorn)
+-   illagrenan (Vašek Dohnal)
+-   browniebroke (Bruno Alla)
+-   ahmed-cheikh (Ahmed Cheikh)
+-   jschneier (Josh Schneier)
+-   brianmay (Brian May)
+-   williamwang61 (Will Wang)
+-   adamcharnock (Adam Charnock)
+-   paveltyavin (Pavel)
+-   jameshiew (James Hiew)
+-   mgrdcm (Dan Moore)
+-   korkmaz (Ahmet Korkmaz)
+-   petrmifek
+-   ryan-copperleaf (Ryan O’Hara)
+-   gatsinski (Hristo Gatsinski)
+-   raghavsethi (Raghav Sethi)
+-   jdufresne (Jon Dufresne)
+-   trik (Marco Marche)
+-   krishraghuram (Raghuram Krishnaswami)
+-   int_ua (Serhiy Zahoriya)
+-   Marpop (Marcin Popławski)
+-   frnhr (Fran Hrženjak)
+-   gelbander (Gustaf Elbander)
+-   dfirst
+-   lorenzomorandini
+-   piotr-szpetkowski (Piotr Szpetkowski)
+-   pacotole (Pacotole)
+-   KenyaDonDraper
+-   andrew-bro (Andrei Loskutov)
+-   toivomattila (Toivo Mattila)
+-   ZuluPro (Anthony Monthe)
+-   kunal15595 (Kunal Khandelwal)
+-   fatanugraha (Fata Nugraha)
+-   aniav (Ania Warzecha)
+-   ababic (Andy Babic)
+-   BramManuel (Bram Janssen)
+-   mofe (Moritz Federspiel)
+-   kjpc-tech (Kyle)
+-   Matthew Hegarty
+-   jinmay (jinmyeong Cho)
+-   DonQueso89 (Kees van Ekeren)
+-   yazdanRa (Yazdan Ranjbar)
+-   Gabriel Warkentin
+-   striveforbest (Alex Zagoro)
+-   josx (José Luis Di Biase)
+-   Jan Rydzewski
+-   rpsands (Ryan P. Sands)
+-   2ykwang (Yeongkwang Yang)
+-   KamilRizatdinov (Kamil Rizatdinov)
+-   Mark Walker
+-   shimakaze-git
+-   frgmt
+-   vanschelven (Klaas van Schelven)
+-   HaPyTeX (Willem Van Onsem)
+-   nikhaldi (Nik Haldimann)
+-   TheRealVizard (Eduardo Leyva)
+-   1gni5 (Jules Ducange)
+-   mpasternak (Michał Pasternak)
+-   nikatlas (Nikos Atlas)
+-   cocorocho (Erkan Çoban)
+-   bdnettleton (Brian Nettleton)
+-   Ptosiek (Antonin)
+-   samupl (Jakub Szafrański)
+-   smunoz-ml (Santiago Muñoz)
+-   carlosal0ns0 (Carlos Alonso)
+-   travenin (Lauri Virtanen)
+-   christophehenry (Christophe Henry)
+-   bgelov (Oleg Belov)
+-   EricOuma (Eric Ouma)
+-   Glay00 (Gleb Gorelov)
```

Comment:
It looks like you have modified all lines here, you should only add your name on a new line.  Please can you revert and submit again.